package.json

```diff
@@ -867,6 +863,16 @@
                 }
             ],
             "commandPalette": [
+                {
+                    "command": "jupyter.notebookeditor.runallcellsabove",
+                    "title": "%DataScience.runAbove%",
+                    "when": "false"
+                },
+                {
+                    "command": "jupyter.notebookeditor.runcellandallbelow",
+                    "title": "%DataScience.runBelow%",
+                    "when": "false"
```

Comment:
It doesn't make a lot of sense to me to have cell level commands on the command palette. It could work with the selected cell if there is one, but as a user I don't think it makes sense. That's why I removed them.

Let me know if you disagree.